packages/legend-graph/src/models/protocols/pure/v1/transformation/pureProtocol/serializationHelpers/V1_RuntimeSerializationHelper.ts

```diff
@@ -71,40 +74,43 @@ export const V1_legacyRuntimeModelSchema = createModelSchema(V1_LegacyRuntime, {
   ),
 });
 
-export const V1_identifiedConnectionModelSchema = createModelSchema(
-  V1_IdentifiedConnection,
-  {
+export const V1_identifiedConnectionModelSchema = (
+  plugins: PureProtocolProcessorPlugin[],
+): ModelSchema<V1_IdentifiedConnection> =>
+  createModelSchema(V1_IdentifiedConnection, {
     connection: custom(
-      (val) => V1_serializeConnectionValue(val, true),
-      (val) => V1_deserializeConnectionValue(val, true),
+      (val) => V1_serializeConnectionValue(val, true, plugins),
+      (val) => V1_deserializeConnectionValue(val, true, plugins),
     ),
     id: primitive(),
-  },
-);
+  });
 
-export const V1_storeConnectionModelSchema = createModelSchema(
-  V1_StoreConnections,
-  {
+export const V1_storeConnectionModelSchema = (
+  plugins: PureProtocolProcessorPlugin[],
+): ModelSchema<V1_StoreConnections> =>
+  createModelSchema(V1_StoreConnections, {
     store: usingModelSchema(V1_packageableElementPointerDeserrializerSchema),
     storeConnections: list(
-      usingModelSchema(V1_identifiedConnectionModelSchema),
+      usingModelSchema(V1_identifiedConnectionModelSchema(plugins)),
     ),
-  },
-);
+  });
 
-export const V1_engineRuntimeModelSchema = createModelSchema(V1_EngineRuntime, {
-  _type: usingConstantValueSchema(V1_RuntimeType.ENGINE_RUNTIME),
-  connections: list(usingModelSchema(V1_storeConnectionModelSchema)),
-  mappings: list(
-    usingModelSchema(V1_packageableElementPointerDeserrializerSchema),
-  ),
-});
+export const V1_setupEngineRuntimeModelSchema = (
```

Comment:
please rename this to `V1_setupEngineRuntimeSerialization` to be consistent with `database` case